Could you please handle the error in curlpp:x64-windows-static and curlpp:x64-windows-static-md, the error as below:
```
CMake Error at scripts/cmake/vcpkg_replace_string.cmake:12 (file):
  file failed to open for reading (No such file or directory):

    D:/packages/curlpp_x64-windows-static/bin/curlpp-config
```

--- a/ports/curlpp/portfile.cmake
+++ b/ports/curlpp/portfile.cmake
@@ -26,7 +26,7 @@ if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
     )
 endif()
 
-if(VCPKG_TARGET_IS_WINDOWS)
+if(VCPKG_TARGET_IS_WINDOWS AND NOT (VCPKG_LIBRARY_LINKAGE STREQUAL static))
     vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/bin/curlpp-config" "${CURRENT_PACKAGES_DIR}" "`dirname $0`/..")
     if(NOT VCPKG_BUILD_TYPE)
         vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/debug/bin/curlpp-config" "${CURRENT_PACKAGES_DIR}" "`dirname $0`/../..")

--- a/versions/c-/curlpp.json
+++ b/versions/c-/curlpp.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "32d1fc0ce976b93900a1aa01b23c0b5ffb2021db",
+      "git-tree": "ebde1c1d303f750ce06e530295a820787878e6cd",
       "version-date": "2018-06-15",
       "port-version": 7
     },

